python/lsst/ip/diffim/dcrModel.py

```diff
@@ -150,6 +150,44 @@ def fromDataRef(cls, dataRef, datasetType="dcrCoadd", numSubfilters=None, **kwar
             modelImages.append(dcrCoadd.image)
         return cls(modelImages, filterInfo, psf, mask, variance, photoCalib)
 
+    @classmethod
+    def fromQuantum(cls, availableCoaddRefs):
+        """Load an existing DcrModel from a Gen 3 repository.
+
+        Parameters
+        ----------
+        availableCoaddRefs : `dict` of `int` : `lsst.daf.butler.DeferredDatasetHandle`
+            Dictionary of spatially relevant retrieved coadd patches,
+            indexed by their sequential patch number.
+
+        Returns
+        -------
+        dcrModel : `lsst.pipe.tasks.DcrModel`
+            Best fit model of the true sky after correcting chromatic effects.
+        """
+        filterInfo = None
+        psf = None
+        mask = None
+        variance = None
+        photoCalib = None
+        modelImages = [[] for _s in range(len(availableCoaddRefs))]
```

Comment:
There should be exactly one image per subfilter in a stored model shouldn't be ?  I'd recommend to initialize `modelImages` with a list of `None`-s as they are simply replaced later. What about `[None, ] * len(availableCoaddRefs)` ?